src/main/java/ATM.java

```diff
@@ -0,0 +1,25 @@
+package src.main.java;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ATM {
+	Map<String,ATMaccount> accounts = new HashMap<>();
+	private Application app;
+	public ATM(Application app) {
```

Comment:
Ideally, your class should have one responsibility. If you need to send the `Application` class to the `ATM` class as an argument to its function/constructor because you need the `Application` class to execute some behavior inside of your `ATM` class, something is wrong design wise. Further, you use `app` to execute `makeAccount()` method, which is again an anti-pattern. The `Application` class should handle only responsibilities of the application, not of the `Account`. Try to imagine this in a different way, and split behavior based on class responsibility. I let you reimagine this scenario and refactor the code accordingly in order to respect these principles.

You might want to check these clean code principles [here](https://www.boldare.com/blog/kiss-yagni-dry-principles/) and [here](https://medium.com/@curiousraj/the-principles-of-clean-code-dry-kiss-and-yagni-f973aa95fc4d) and try to refactor the code in order to respect them as much as possible.